inst/doc/embed.html

```diff
@@ -680,9 +680,9 @@ <h2>Easy but Ugly</h2>
+<pre><div class="diffobj-row"><div class="line banner"><div class="delete"><div class="gutter"><div class="delete">&lt;</div><div class="pad"> </div></div><div class="text"><div class="delete"><span class="word delete">1:5</span></div></div></div></div><div class="line banner"><div class="insert"><div class="gutter"><div class="insert">&gt;</div><div class="pad"> </div></div><div class="text"><div class="insert"><span class="word insert">2:6</span></div></div></div></div></div><div class="diffobj-row"><div class="line"><div class="header">@@ 1 @@</div></div><div class="line"><div class="header">@@ 1 @@</div></div></div><div class="diffobj-row"><div class="line"><div class="delete"><div class="gutter"><div class="delete">&lt;</div><div class="pad"> </div></div><div class="text"><div class="delete"><span class="trim">[1] </span><span class="word delete">1</span> 2 3 4 5<span class="trim"></span></div></div></div></div><div class="line"><div class="insert"><div class="gutter"><div class="insert">&gt;</div><div class="pad"> </div></div><div class="text"><div class="insert"><span class="trim">[1] </span>2 3 4 5 <span class="word insert">6</span><span class="trim"></span></div></div></div></div></div></pre>
 </div>
-<p>This is an ugly implementation because it produces illegal HTML. The styles are directly embedded in the body of the document, outside of the HEAD tags. Although this is illegal HTML, it seems to work in most browsers. Another problem is that every diff you use in your document will inject the same CSS code over and over.</p>
+<p>This is an ugly implementation because it produces illegal HTML. The styles are directly embedded in the body of the document, outside of the HEAD tags. Although this is illegal HTML, it seems to work in most bdiffobj-rowsers. Another problem is that every diff you use in your document will inject the same CSS code over and over.</p>
```

Comment:
Looks like a case of over-eager search-and-replace here.

Reply:
Actually, this doesn't matter since you did not commit the vignette source itself.